tests/templates/test_embeddings/test_qaoa_emb.py

```diff
@@ -262,25 +262,6 @@ class TestInterfaces:
     """Tests that the template is compatible with all interfaces, including the computation
     of gradients."""
 
-    def test_list_and_tuples(self, tol):
-        """Tests common iterables as inputs."""
```

Comment:
as much as the new system makes sense, I will miss being able to support this :(

Reply:
It's not that it is no longer possible at all to pass lists and tuples to QNodes. As long as they are for non-trainable arguments (which in principle is the case here), it is still supported.
The reason this test (and other similar tests) were removed is that they use array slicing *and* attempt to pass lists/tuples to functions which only work on arrays (`QAOAEmbedding` in this case). Thus, this test scenario really doesn't make sense except in the context of testing the implicit conversion of QNode arguments (which was removed).

Reply:
Got it!